### PR TITLE
Fix browser example

### PR DESCRIPTION
**Pull request by relu91**
This PR introduces back the browser bundle among the supported packages. A couple of notes:
1. I discovered that `node-fetch` when used in browsers defaults to native `fetch` implementation this means the http client was sending `ReadableStreams` (instead of `Readable` from NodeJS) instance to the core. The core is expecting that every binding respect the contract of the `Content` interface which requires a NodeJS `Readable`. This PR changes `binding-http` to convert the stream before sending it the the `core`
2. `browserify` is using an old package for replacing nodejs `url` package. This creates issues when directly importing `url`. I discovered that using the global `URL` variable bypass `browserify` and it works well in browsers too. This PRs adds this workaround. Maybe #640 is another possible solution
3. I improved a little bit the example code and updated it
4. I fixed a bug when handling HTTP log polling requests from events. The servent was not handling head requests properly.

### CODE REVIEW

File: examples/browser/counter.js [hunk 1]
```diff
@@ -44,12 +44,12 @@ function showInteractions(thing) {
             dtItem.appendChild(link);
             document.getElementById("properties").appendChild(dtItem);
             document.getElementById("properties").appendChild(ddItem);
-
+            console.log("property: " + td.properties[property]);
```
Comment (relu91) [inline examples/browser/counter.js:47]: Just noticed this left over, I'll update the pr after CI gives green light

**Comment by danielpeintner** [comment]
Thanks @relu91 👍 

Note: No review, I still need to take a look. I just want to mention that there are 2 more "browser" examples we need to work on
* index
* smart-coffee-machine

We can tackle them in another PR. As you like @relu91?

**Comment by relu91** [comment]
Oh yeah, you're right I think we can do that in another PR!

**Review by danielpeintner — CHANGES_REQUESTED** [review]
I did run into some issues on my side
* a global `npm run build` does NOT create the dist folder for browser-bundle.
One needs to walk into packages\browser-bundle and call 
`npm run build` there to get `wot-bundle.min.js`
* When trying to run the counter.html I get several errors and the UI does not show
  - Uncaught TypeError: class heritage _$core_31.default is not an object or null
  - Uncaught ReferenceError: Wot is not defined

grafik

**Comment by relu91** [comment]
> a global npm run build does NOT create the dist folder for browser-bundle.
One needs to walk into packages\browser-bundle and call
npm run build there to get wot-bundle.min.js

Nice catch I didn't check for that sorry. Since the `browser-bundle` is not a TS project I've added the relevant commend in `build`.

> When trying to run the counter.html I get several errors and the UI does not show
Uncaught TypeError: class heritage _$core_31.default is not an object or null
Uncaught ReferenceError: Wot is not defined

It seems that is a `tinify` error... I temporally remove it but I think we should file an issue so that in the future we can understand what's going on.

**Comment by danielpeintner** [comment]
> It seems that is a `tinify` error... I temporally remove it but I think we should file an issue so that in the future we can understand what's going on.

https://github.com/eclipse/thingweb.node-wot/issues/690 created

**Review by danielpeintner — APPROVED** [review]
Looks good!

Minor: do you want to remove the console.log ?

**Pull request merged**